modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -153,6 +153,14 @@ These adjustments are only applied to cleartext, edge-terminated, and re-encrypt
 
 For request headers, these adjustments are applied only for routes that have the `haproxy.router.openshift.io/h1-adjust-case=true` annotation. For response headers, these adjustments are applied to all HTTP responses. If this field is empty, no request headers are adjusted.
 
+|`tuningOptions`
+|`tuningOptions` specifies options for tuning the performance of Ingress Controller pods.
+
+* `headerBufferBytes` specifies how much memory is reserved in bytes for Ingress Controller connection sessions. This value must be at least `16384` if HTTP/2 is enabled for the Ingress Controller. If this field is empty, the Ingress Controller uses a default value of `32768` bytes. This field is optional and not recommended because `headerBufferBytes` values that are too small can break the Ingress Controller, and `headerBufferBytes` values that are too large could cause the Ingress Controller to use significantly more memory than necessary.
+
+* `headerBufferMaxRewriteBytes` specifies the number of threads created per HAProxy process. Creating more threads allows each Ingress Controller pod to handle more connections, at the cost of more system resources being used. HAProxy
```

Comment:
Hm, this paragraph is describing the new `threadCount` field.
I think there should be a total of 3 paragraphs here?

Reply:
The other header buffer field is `headerBufferMaxRewriteBytes` https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go#L1063

Reply:
Ah thank you! I'll get that updated. I was a bit confused reading through the API.

Reply:
Updated! And I updated lower-case references to [Ingress] Controller for consistency in the file.